Models/articles-model.js

```diff
@@ -28,3 +28,18 @@ exports.selectArticleById = id => {
       return rows[0];
     });
 };
+
+exports.updateArticleById = (id, votes) => {
+  const { inc_votes } = votes;
+  const queryString = `UPDATE articles
+                       SET votes = votes + $1
+                       WHERE article_id = $2
+                       RETURNING *;`;
```

Comment:
Great job setting the votes in your SQL query and using a parameterised query to protect against SQL injection